testing/coreruleset/coreruleset_test.go

```diff
@@ -53,7 +53,7 @@ func BenchmarkCRSCompilation(b *testing.B) {
 	}
 }
 
-func BenchmarkCRSSimpleGET(b *testing.B) {
+func BenchmarkCRSRuntimeSimpleGET(b *testing.B) {
```

Comment:
I'm not too clear on what Runtime means here. If it means running time is the operation, I think all benchmarks measure that so doesn't need to be in name